FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -843,77 +854,56 @@ OrderedNode *OpDispatchBuilder::SelectCC(uint8_t OP, IR::OpSize ResultSize, Orde
 
   switch (OP) {
     case 0x0: { // JO - Jump if OF == 1
-      auto Flag = GetRFLAG(FEXCore::X86State::RFLAG_OF_LOC);
-      SrcCond = _Select(ResultSize, OpSize::i32Bit, CondClassType{COND_NEQ},
-          Flag, ZeroConst, TrueValue, FalseValue);
+      SrcCond = SelectNZCV(FEXCore::X86State::RFLAG_OF_LOC, 1, ResultSize, TrueValue, FalseValue);
```

Comment:
A minor nit, but maybe prefer using the boolean literal equivalent to make it evident at the call site that the parameter is a boolean, rather than an integer type

```suggestion
      SrcCond = SelectNZCV(FEXCore::X86State::RFLAG_OF_LOC, true, ResultSize, TrueValue, FalseValue);
```

ditto for the others.

Reply:
done